en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md

```diff
@@ -0,0 +1,340 @@
+    ```
+
+    You will receive the following response:
+
+    ```bash
+    {"Hello":"World"}%
+    ```
+
+### Invoke without Ingress controller
+
+Once you have [deployed your integrations](#deploy-integration-solutions-in-k8s) in the Kubernetes cluster, you can also invoke the integration solutions without going through the Ingress controller by using the [port-forward](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/#forward-a-local-port-to-a-port-on-the-pod) method for services. 
+
+Follow the steps given below:
+
+1. Apply port forwarding:
+
+    ```bash
+    kubectl port-forward service/hello-world-service 8290:8290
+    ```
+
+2. Invoke the proxy service:
+
+    ```bash
+    curl http://localhost:8290/services/HelloWorld
+    ```
+
+    You will receive the following response:
+
+    ```bash
+    {"Hello":"World"}%
+    ```
+
+## Update existing integration deployments
+
+The EI K8s operator allows you to update the Docker images used in Kubernetes pods(replicas) with the latest update of the tag. To pull the latest tag, we need to delete the associated pod with its pod ID as follows:
+
+```bash 
+kubectl delete pod <POD-NAME>
+```
+
+When you run the above command, Kubernetes will spawn another temporary pod, which has the same behaviour of the pod we have deleted. Then the deleted pod will restart by pulling the latest tag from the Docker image path. 
+
+!!! Note 
+    Here we are recommending to use a different image path for the updated integration solution. Otherwise, (because the Docker image is re-pulled from the existing deployment) some traffic from outside might get lost. 
+
+
+## Run inbound endpoints
+
+[Inbound Endpoints](../../../references/synapse-properties/inbound-endpoints/about-inbound-endpoints/) in the Micro Integrator are used for separating endpoint listeners. That is, for each HTTP inbound endpoint, messages are handled separately. Also, we can create any number of inbound endpoints on any port. 
+
+Therefore, we can expose the inbound endpoint ports from the Kubernetes cluster by passing the `inboundPorts` property inside our `integration_cr.yaml` custom resource file as follows:
+
+```yaml
+apiVersion: "integration.wso2.com/v1alpha2"
+kind: "Integration"
+metadata:
+  name: "inbound-samples"
+spec:
+  image: "<Docker image for the inbound endpoint Scenario>"
+  deploySpec:
+    minReplicas: 1
+  expose:
+    passthroPort: 8290
+    inboundPorts:
+      - 8000
+      - 9000
+      ...
+```
+
+Use the following methods to invoke the inbound endpoints in HTTP and HTTPS transports. Note that `<INTEGRATION-NAME>` is the value we have used as the metadata name in the `integration_cr.yaml` file. 
+
+-   HTTP request
+
+    ```bash
+    curl http://<HOST-NAME>/<INTEGRATION-NAME>-inbound/<PORT>/<CONTEXT>
+    ```
+
+-   HTTPS request
+
+    ```bash
+    curl --cacert <CERT_FILE> https://<HOST-NAME>/<INTEGRATION-NAME>-inbound/<PORT>/<CONTEXT>
+    ```    
```

Comment:
It worth mentioning below. 

- all ingress controller related configs are controlled by `integration-ingress-config` configMap defined at `integration_controller_conf.yaml` file. 
- If `minReplicas` and `maxReplicas` are not defined in `` the operator will get pick up what is defined at `integration_controller_conf.yaml` file as defaults. 
- If `autoScale. enabled` is not defined at `` the operator will get pick up what is defined at `integration_controller_conf.yaml` file as the default. 
- All Horizontal Pod scaling configurations are defined under `hpaMetrics` at `integration_controller_conf.yaml` file.

Finally, 
- You can update the `reconcileInterval` (how frequent operator should scan for artefact updates) at `integration_controller_conf.yaml` file.